lib/event.rb

```diff
@@ -0,0 +1,38 @@
+require './lib/craft'
+require './lib/person'
+
+class Event
+
+  attr_reader :name, :crafts, :attendees
+
+  def initialize(event_name, crafts, attendees)
+    @name = event_name
+    @crafts = crafts
+    @attendees = attendees
+  end
+
+  def attendee_names
+    @attendees.map { |attendee| attendee.name }
+  end
+
+  def craft_with_most_supplies
+    @crafts.max_by { |craft| craft.supplies_required.length }.name
+  end
+
+  def supply_list
+    crafts.flat_map { |craft| craft.supplies_required.keys }.uniq
```

Comment:
We are expecting strings to be returned here so: 
```suggestion
    crafts.flat_map { |craft| craft.supplies_required.keys.to_s }.uniq
```